cosmos-gui/public/stylesheets/style.css

```diff
@@ -156,6 +156,12 @@ header {
   padding: 0px 10px;
   margin-bottom: 10px;
 }
+.sidebar a {
```

Comment:
I guess the site or portal is not responsive, right? I mean, using absolute units as pixels would make the site to be badly seen for certain resolutions... :) Relative units such as em or rem should be used (http://www.w3schools.com/cssref/css_units.asp), but I leave it totally to your discretion... :)